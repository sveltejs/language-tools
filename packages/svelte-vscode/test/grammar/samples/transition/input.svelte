<div transition:fade></div>

<div in:fade></div>

<div out:fade></div>
