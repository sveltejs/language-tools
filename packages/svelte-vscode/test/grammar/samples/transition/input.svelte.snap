><div transition:fade></div>
#^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte punctuation.definition.tag.begin.svelte
# ^^^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte entity.name.tag.svelte
#    ^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte
#     ^^^^^^^^^^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte meta.directive.transition.svelte keyword.other.animation.svelte
#               ^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte meta.directive.transition.svelte punctuation.definition.keyword.svelte
#                ^^^^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte meta.directive.transition.svelte entity.name.type.svelte
#                    ^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte punctuation.definition.tag.end.svelte
#                     ^^ source.svelte meta.scope.tag.div.svelte meta.tag.end.svelte punctuation.definition.tag.begin.svelte
#                       ^^^ source.svelte meta.scope.tag.div.svelte meta.tag.end.svelte entity.name.tag.svelte
#                          ^ source.svelte meta.scope.tag.div.svelte meta.tag.end.svelte punctuation.definition.tag.end.svelte
>
><div in:fade></div>
#^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte punctuation.definition.tag.begin.svelte
# ^^^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte entity.name.tag.svelte
#    ^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte
#     ^^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte meta.directive.in.svelte keyword.other.animation.svelte
#       ^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte meta.directive.in.svelte punctuation.definition.keyword.svelte
#        ^^^^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte meta.directive.in.svelte variable.function.svelte
#            ^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte punctuation.definition.tag.end.svelte
#             ^^ source.svelte meta.scope.tag.div.svelte meta.tag.end.svelte punctuation.definition.tag.begin.svelte
#               ^^^ source.svelte meta.scope.tag.div.svelte meta.tag.end.svelte entity.name.tag.svelte
#                  ^ source.svelte meta.scope.tag.div.svelte meta.tag.end.svelte punctuation.definition.tag.end.svelte
>
><div out:fade></div>
#^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte punctuation.definition.tag.begin.svelte
# ^^^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte entity.name.tag.svelte
#    ^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte
#     ^^^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte meta.directive.out.svelte keyword.other.animation.svelte
#        ^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte meta.directive.out.svelte punctuation.definition.keyword.svelte
#         ^^^^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte meta.directive.out.svelte variable.function.svelte
#             ^ source.svelte meta.scope.tag.div.svelte meta.tag.start.svelte punctuation.definition.tag.end.svelte
#              ^^ source.svelte meta.scope.tag.div.svelte meta.tag.end.svelte punctuation.definition.tag.begin.svelte
#                ^^^ source.svelte meta.scope.tag.div.svelte meta.tag.end.svelte entity.name.tag.svelte
#                   ^ source.svelte meta.scope.tag.div.svelte meta.tag.end.svelte punctuation.definition.tag.end.svelte
>